test/check-machines-create

```diff
@@ -1366,6 +1366,7 @@ vnc_password= "{vnc_passwd}"
             root = ET.fromstring(rhel_8_1_xml)
             root.find('os').find('resources').find('minimum').find('ram').text = '134217728'
             root.find('os').find('resources').find('minimum').find('storage').text = '134217728'
+            root.find('os').find('release-date').text = datetime.today().strftime('%Y-%m-%d')
```

Comment:
I think same fix should be added for thel RHEL_8_2, which is being faked just few lines below. Otherwise we will run into the same issue in few months.

Reply:
Done!